app/routes/home.js

```diff
@@ -34,7 +34,17 @@ const books = [
 ];
 export default Ember.Route.extend({
 
-  model() {
+  queryParams: {
```

Comment:
A lot of work to be done by the av-table component user. I understand that we need to trade it off by not using query params.